crush.json

```diff
@@ -1,8 +1,144 @@
 {
   "$schema": "https://charm.land/crush.json",
   "lsp": {
-    "Go": {
+    "go": {
       "command": "gopls"
+    },
+    "gomod": {
+      "command": "gopls"
+    },
+    "gotmpl": {
+      "command": "gopls"
+    },
```

Comment:
the map key is not the file format, its just a name.

this will start 3 gopls processes.